lcli/src/parse_ssz.rs

```diff
@@ -59,7 +59,7 @@ pub fn run_parse_ssz<E: EthSpec>(
         spec.config_name.as_deref().unwrap_or("unknown"),
         E::spec_name()
     );
-    info!("Type: {type_str}");
+    info!(?type_str, "Type");
```

Comment:
not sure if we need a structured field here. 

we can either keep it as
`info!("Type: {type_str}");`

or only use a structured field w/o log message 

i.e.

`info!(type = ?type_str);

wondering if anyone else has opinions about this